packages/eddsa-proof/rollup.browser.config.ts

```diff
@@ -0,0 +1,38 @@
+import alias from "@rollup/plugin-alias"
+import json from "@rollup/plugin-json"
+import * as fs from "fs"
+import cleanup from "rollup-plugin-cleanup"
+import typescript from "rollup-plugin-typescript2"
+
+const pkg = JSON.parse(fs.readFileSync("./package.json", "utf-8"))
+const banner = `/**
+ * @module ${pkg.name}
+ * @version ${pkg.version}
+ * @file ${pkg.description}
+ * @copyright Ethereum Foundation 2023
```

Comment:
```suggestion
 * @copyright Ethereum Foundation 2024
```

We could use `new Date().getFullYear()` in all the packages.